ui/components/spinner.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export default function Spinner({ height }) {
+  const borderWidth = Math.ceil(height / 10);
+  const indicatorStyle = { height: `${height}px` };
```

Comment:
Agreed, that would be wonderful. I hacked around a little with 100% width/height and setting max values on containers, but had two issues that I didn't know how to get around:

1. The indicator _must_ be a square, otherwise it looks like a wobbly oval
2. The divs we're loading (selected filters, main page content) are arbitrarily tall

Hopefully this is easy enough to play with now; maybe using a gif would solve those issues?